packages/cojson/src/PeerState.ts

```diff
@@ -226,6 +226,7 @@ export class PeerState {
       peerId: this.id,
       peerRole: this.role,
     });
+    this.peer.crashOnClose = false;
```

Comment:
This fixes some errors I've started to get when closing nodes with a storage peer.

Don't know why it didn't happen before